core/templates/services/UpgradedServices.ts

```diff
@@ -756,6 +770,12 @@ export class UpgradedServices {
     upgradedServices['ExplorationStatsObjectFactory'] =
       new ExplorationStatsObjectFactory(
         upgradedServices['StateStatsObjectFactory']);
+    upgradedServices['ExplorationTaskObjectFactory'] =
```

Comment:
This needs to be added to oppia-angular-root.component.ts. Please follow point number 12 under this: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services.

Reply:
Done!

Reply:
@brianrodri this hasn't been done.
Make sure that you have:

1. imported the service.
2. created a static class variable (use camelCase of the service class name as the static variable name).
3. added the service to the constructor.
4. assigned the service to the static class variable in ngAfterViewInit

You have missed checks 3 and 4.

Reply:
Ah I see, my mistake. Done now.